airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -1007,6 +1007,16 @@ def _optional_boolean(value: bool | None) -> bool | None:
         )
     ),
 ]
+QueryHITLDetailMapIndexFilter = Annotated[
+    FilterParam[str | None],
```

Comment:
Should this be `int` instead of `str`?
```suggestion
    FilterParam[int | None],
```

Reply:
Yes, thanks for the reminder